framework/analytics/__init__.py

```diff
@@ -29,49 +23,14 @@
 @run_postcommit(once_per_request=False, celery=True)
 @app.task(max_retries=5, default_retry_delay=60)
 def increment_user_activity_counters(user_id, action, date_string, db=None):
-    db = db or database  # default to local proxy
-    if not db and settings.DEBUG_MODE:
-        logger.warn('Cannot connect to database. Analytics will be unavailable')
-        return
-    collection = database['useractivitycounters']
-    date = parser.parse(date_string).strftime('%Y/%m/%d')
-    query = {
-        '$inc': {
-            'total': 1,
-            'date.{0}.total'.format(date): 1,
-            'action.{0}.total'.format(action): 1,
-            'action.{0}.date.{1}'.format(action, date): 1,
-        }
-    }
-    collection.update(
-        {'_id': user_id},
-        query,
-        upsert=True,
-        manipulate=False,
-    )
-    return True
+    from osf.models import UserActivityCounter
+    # NOTE maybe this should act differently in DEBUG_MODE, I disagree
```

Comment:
I also disagree; that DEBUG_MODE check was there just so I could get the app to run locally while this stuff wasn't implemented yet. So this is good as is.